dme/resource_dme_record.go

```diff
@@ -195,15 +195,15 @@ func getAll(d *schema.ResourceData, cr map[string]interface{}) error {
 	if attr, ok := d.GetOk("value"); ok {
 		cr["value"] = attr.(string)
 	}
-	if attr, ok := d.GetOk("gtdLocation"); ok {
+	if attr, ok := d.GetOk("gtd_location"); ok {
 		cr["gtdLocation"] = attr.(string)
```

Comment:
it make look suspicious that `cr["gtdLocation"]` wasn't fixed as well. This is because the way it shows in the `cr` map is the way it gets encoded to JSON, and then sent to dnsme. In order to preserve functionality, it should remain as is here. It only needed to be changed on the terraform schema side.

Reply:
also the reasoning behind the field renaming is because otherwise you get this error:
https://github.com/hashicorp/terraform/blob/master/helper/schema/schema.go#L840